### DISCUSSION TITLE
Welcome to Stewardess Discussions!

### DISCUSSION THREAD

**Discussion by cmp-nct** [cmp-nct/Stewardess#1] [Announcements]
## 👋 Welcome!
The goal of this plugin is to repair errors in Github's copilot behavior, contribution, discussions, ideas always welcome.

**Comment by mavanb** [comment]
Any plans to make it available to the other Jetbrains products?

**Reply by cmp-nct** [reply]
I believe it will already work for the typical Jetbrains IDE, not just PHPStorm.
Just install the live plugin extension, then load my code.

There are a few smaller parsing flaws (when completing the last char that is a ] or a '\n' it sometimes fails.
But it works well enough for me to not have fixed that yet.

**Comment by CodenameCookie** [comment]
This is brilliant. I'm super surprised more people are not using it!

**Reply by cmp-nct** [reply]
At the time of writing this I did not expect it to survive long, the flaws in the official extensions can be fixed in a few hours by the maintainers.
Soon a year has passed and they still do not plan to improve it.
